service/prometheus/update.go

```diff
@@ -31,20 +31,6 @@ func UpdateConfig(promcfg config.Config, scrapeConfigs []config.ScrapeConfig) (c
 // isManaged returns true if the given scrape config is managed by the prometheus-config-controller,
 // false otherwise.
 func isManaged(scrapeConfig config.ScrapeConfig) bool {
-	// TODO: the static config detection can be removed once all installations use k8s sd.
-	for _, targetGroup := range scrapeConfig.ServiceDiscoveryConfig.StaticConfigs {
-		if _, ok := targetGroup.Labels[ClusterLabel]; ok {
-			return true
-		}
-	}
-
-	// TODO: the clusterlabel can be removed once all installations use clusteridlabels.
-	for _, relabelConfig := range scrapeConfig.RelabelConfigs {
-		if relabelConfig.TargetLabel == ClusterLabel {
-			return true
```

Comment:
This is removing support for checking if the `prometheus_config_controller` label is present, which is the older method for checking if a scrape config is managed by the `prometheus-config-controller`.